tcms/core/migrations/0001_django_comments__object_pk.py

```diff
@@ -4,16 +4,41 @@
 from django.db import migrations, models
 
 
+class DjangoCommentsAlterField(migrations.AlterField):
+    """
+        Custom migration operation which alters the `django_comments`
```

Comment:
I'd like to keep docstring in this format throughout Nitrate.

```
"""summary

More description (could be multiple lines)
"""
```

Please update this dostring.